url-updater/src/main/java/com/devonfw/tools/ide/url/updater/AbstractUrlUpdater.java

```diff
@@ -38,6 +38,7 @@
 import com.devonfw.tools.ide.url.model.folder.UrlTool;
 import com.devonfw.tools.ide.url.model.folder.UrlVersion;
 import com.devonfw.tools.ide.url.model.report.UrlUpdaterReport;
+import com.devonfw.tools.ide.url.tool.pip.PipUrlUpdater;
```

Comment:
A best practice I learned once from checkstyle is that imports should only be used for productive code and not for JavaDoc only.
If the referenced class (`PipUrlUpdater`) is not present then also loading this class (`AbstractUrlUpdater`) cannot be loaded at runtime.
Therefore if we have JavaDoc links to classes not otherwise referenced in the class we should use fully qualified references instead. IMHO IntelliJ refactoring is too stupid for this (Eclipse handles this properly)...

```suggestion
```

Reply:
Replaced with FQR now.